services/types.go

```diff
@@ -369,6 +383,5 @@ func (t *transaction) UnmarshalJSON(data []byte) error {
 	t.GasPrice = gasPrice
 	t.GasLimit = gasLimit
 	t.ChainID = chainID
-	t.GasPrice = gasPrice
```

Comment:
not sure I understand why you are removing this line?

Reply:
looks like it may have been a dupe :)